src/components/CodeView/index.tsx

```diff
@@ -72,7 +72,17 @@ export class CodeViewBase extends React.Component<Props> {
       <React.Fragment>
         {selectedMessageMap &&
           selectedMessageMap.global.map((message) => {
-            return <LinterMessage key={message.uid} message={message} />;
+            return (
+              <LinterMessage
+                key={message.uid}
+                message={message}
+                _scrollToElement={
```

Comment:
Do you really need to patch `CodeView` for this? This won't work on the compare page since `CodeView` is not rendered there. 

Instead of passing a `_scrollToElement` prop, would it be possible to connect `LinterMessage` to Redux state and have it look in state to know if it should be scrolled into view?

Reply:
Yes, I'm going to try using Redux state instead.